src/templatizer/templatizer.html

```diff
@@ -39,6 +39,15 @@
         new DataTemplate();
         PatchedHTMLTemplateElement._newInstance = null;
       }
+      _flushProperties() {
```

Comment:
Revised gambit: Don't flush wrapper early, instead squelch forwarding based on `if` and add logic for capturing invalidated properties while `if` is false and re-syncing them upon `true`.